src/common.ts

```diff
@@ -110,3 +110,19 @@ export type UpdateRegistrantStatusBody = {
 export type UpdateRegistrantStatusParams = {
   occurrence_id?: string;
 };
+
+export type ZoomMeetingRecordingFileType =
+  'MP4' | 'M4A' | 'TIMELINE' | 'TRANSCRIPT' | 'CHAT' | 'CC';
+
+export type ZoomMeetingRecordingType =
```

Comment:
From the [docs](https://marketplace.zoom.us/docs/api-reference/zoom-api/cloud-recording/recordingslist), `TIMELINE` isn't a type and `poll` is missing here.